src/components/static/ToggleSelection.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useState } from "react";
+import Arrows from "../../../public/faqArrows.svg";
+import Image from "next/image";
+
+const ToggleSelection = ({ title, children }) => {
+  const [state, setState] = useState(false);
+  return (
+    <div className="bg-bear-dark py-2">
+      <button
+        onClick={() => setState(!state)}
+        className="flex items-center justify-between w-full py-2 pr-8 text-white font-chonburi text-sm md:text-xl"
+      >
+        {title}
+        <Image
+          src={Arrows}
+          width={25}
+          className={`${state && "rotate-180"} duration-300`}
+        />
+      </button>
+      {!state && (
+        <div className="flex flex-row justify-end">
+          <div className="h-[1px] w-full bg-white"> </div>
+          <div className="w-2 h-2 bg-white transform rotate-45 absolute flex -mt-1"></div>
+        </div>
+      )}
+      {state && (
+        <div className="py-4 text-white text-xs md:text-lg font-cormorant">
```

Comment:
Add a transition on FAQ expansion?

Example with transition using max-height padding-y and opacity. Though it's a bit rough with the click hitbox (maybe change button order so it has higher priority?) and you can tune the duration of it as it seems pretty fast.
```jsx
<div className={"text-white text-xs md:text-lg font-cormorant transition-all " + (state ? "max-h-96 py-4 opacity-100" : "max-h-0 py-0 opacity-0")}>
  {children}
</div>
```
![bh-exp](https://github.com/bearhack-ucr/bearhack2024/assets/12877445/f03104f9-d4f3-48dc-9e61-d939d459fa46)